_includes/sidebar.html

```diff
@@ -1,39 +1,53 @@
-{% include custom/sidebarconfigs.html %}
-
-<ul id="mysidebar" class="nav affix">
-    <li class="sidebarTitle">{{sidebar[0].product}} {{sidebar[0].version}}</li>
-    {% for entry in sidebar %}
-    {% for folder in entry.folders %}
-    {% if folder.output contains "web" %}
-      {% if folder.external_url %}
-      <li>
-        <a href="{{folder.external_url}}" target="_blank">{{ folder.title }}</a>
-      </li>
-      {% elsif folder.spacer %}
-      <br>
-      {% else %}
-      <li>
-        <a href="{{folder.url}}">{{ folder.title }}</a>
-
-        {% if folder.folderitems %}
-        <ul style="display:none">
-            {% for folderitem in folder.folderitems %}
-            {% if folderitem.output contains "web" %}
-            {% if folderitem.external_url %}
-            <li><a href="{{folderitem.external_url}}" target="_blank">{{folderitem.title}}</a></li>
-            {% elsif page.url == folderitem.url %}
-            <li class="active"><a href="{{folderitem.url}}">{{folderitem.title}}</a></li>
-            {% else %}
-            <li><a href="{{folderitem.url}}">{{folderitem.title}}</a></li>
-            {% endif %}
-            {% endif %}
-            {% endfor %}
-        </ul>
-        {% endif %}
-      </li>
-      {% endif %}
-
-    {% endif %}
-    {% endfor %}
-    {% endfor %}
+<ul id="mysidebar" class="nav">
+
+  <li class="sidebar-title">Get started</li>
+
+    <ul class="sidebar-items">
+      <li><a href="/setup/">Setup and install</a></li>
+      <li><a href="/getting-started/">Create and run your first app</a></li>
+      <li><a href="/bootstrap-into-dart/">Bootstrap into Dart</a></li>
+      <li><a href="/design-principles/">Design principles</a></li>
+      <li><a href="https://docs.google.com/presentation/d/1cw7A4HbvM_Abv320rVgPVGiUP2msVs7tfGbkgdrTy0I/edit?usp=sharing">Architecture diagrams</a></li>
+    </ul>
+
+  <li class="sidebar-title">Build UIs</li>
+
+    <ul class="sidebar-items">
+      <li><a href="/widgets-intro/">Tour the framework</a></li>
+      <li><a href="/widgets/">Widget and component catalog</a></li>
```

Comment:
I'd remove 'and component', that word is fuzzy enough that it confuses more than it clarifies